plugins.json

```diff
@@ -1685,7 +1727,47 @@
       }
     },
     "enterprise": false,
-    "updated_at": "2020-04-22T18:12:27.541293964Z"
+    "updated_at": "2020-04-21T17:20:23Z"
+  },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-todo",
+    "icon_data": "",
+    "download_url": "https://github.com/mattermost/mattermost-plugin-todo/releases/download/v0.1.0/com.mattermost.plugin-todo-0.1.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-todo/releases/tag/v0.1.0",
```

Comment:
Why should this release get added? The marketplace modal will only show the latest release anyway.

Reply:
No special reason. My thought was that it will be added either way after each `make plugins.json` (since I added todo to the default repos), so I prefer to add it now instead of remove it in this pr and any other update on the list of plugins.

Reply:
Could you please only add `v0.3.0` in this PR? I would like to keep the diff as small as possible and avoid changes to the existing releases. Please me know if you need help.